source/MaterialXTest/Render.cpp

```diff
@@ -1624,14 +1590,18 @@ TEST_CASE("Render validation of test suite", "[render]")
 
     // Dump out profiling information
     totalTime.endTimer();
-    printRunLog(profileTimes, options, usedImpls, profilingLog, dependLib, oslShaderGenerator, glslShaderGenerator
-#if defined(MATERIALX_BUILD_GEN_OGSFX)
-        , ogsfxShaderGenerator
+    printRunLog(profileTimes, options, usedImpls, profilingLog, dependLib
+#ifdef MATERIALX_BUILD_GEN_OSL
```

Comment:
There are render build defines which are used now. This is the flag just to say if shader gen was built.
e.g. MATERIALX_BUILD_RENDERGLSL should be used instead of MATERIALX_BUILD_GEN_GLSL.

Reply:
Ah ok, will fix!